loom/common/utils/utByteArray.h

```diff
@@ -32,6 +32,16 @@ class utByteArray {
     utArray<unsigned char> _data;
     UTsize _position;
 
+    void memcpyAligned(void *destination, const void *source, size_t num)
```

Comment:
Usually when you specify alignment for something, it's indicative that it requires that alignment. So in this case, we should say Unaligned or 1 byte alignment to indicate its purpose.